### PR TITLE
[DE] Translation Quote

### PR DESCRIPTION
**Pull request by DerSchlupf**
Forgot the the quote :D 
It seems. that i should break the lines earlier. which latex-command is it? for example: 

"Ich tue dies.... (next line)
und weil ich ..."

new one:
image

old one:
image

### CODE REVIEW

**Comment by DarthGandalf** [comment]
There're a few options, e.g. `\\` (and since it's po, the number of \ doubles to `\\\\`)

https://www.overleaf.com/learn/latex/Line_breaks_and_blank_spaces has more

File: main_de.tex [hunk 1]
```diff
@@ -19,7 +19,7 @@
 \newcommand{\pageshorthand}{S.}
 
 \newcommand{\heegusquote}{
-  \textit{I’m doing this out of the love of learning games and wanting to understand their complexities.}
+  \textit{Ich tue dies aus Liebe zum Lernen von Spielen und weil ich ihre Komplexität verstehen möchte.}
```
Comment (DerSchlupf) [inline main_de.tex:22]:
```suggestion
  \textit{Ich tue dies aus Liebe zum Lernen von Spielen\\ und weil ich ihre Komplexität verstehen möchte.}
```

**Review by qwrtln — APPROVED** [review]

**Comment by DerSchlupf** [comment]
I would say the current one looks better it its ready for merge?

**Comment by qwrtln** [comment]
Hit it!

**Pull request merged**